### DISCUSSION TITLE
[Documentation, addition to the FAQ entry] Could a new entry be added as to whether conky supports "clickable text", such as a button?

### DISCUSSION THREAD

**Discussion by rubyFeedback** [brndnmtthws/conky#1321] [Q&A]
Hello there,

It has been quite some months since I last tried conky. I noticed new releases
were made; I haven't yet caught up on any changes.

Conky as idea is great. There are also many awesome conky files, but I found
that i need to create my own conkyrc, because I don't understand what other
people do.

I even managed to add output from ruby scripts, so this part works fine. \o/

What I do not know is whether clickable areas can be used via conky yet,
such as a "button" or a clickable area. It's fine if conky does not support
it yet, or never will, but I would appreciate a FAQ entry for this at
https://github.com/brndnmtthws/conky/wiki/FAQ, so I can understand whether
this may be added, already works, or will not be added.

Thank you for reading and considering.

**Comment by brndnmtthws** [comment]
> What I do not know is whether clickable areas can be used via conky yet,
> such as a "button" or a clickable area.

Conky does not have any mouse interaction features at the moment.
